client/src/components/dashboard/DashboardContent.js

```diff
@@ -12,42 +13,52 @@ function DashboardContent() {
 	const [inputValue, setInputValue] = useState('');
 	const [errorMessage, setErrorMessage] = useState('');
 	const [copiedKey, setCopiedKey] = useState(null);
-	const [keys, setKeys] = useState([
-		{
-			description: 'Demo Key 1',
-			apiKey: 'maohquwnbtpszjqj91myxk',
-			createDate: 'Dec 18, 2023',
-		},
+	const [keys, setKeys] = useState([]);
+
+	const {data, makeRequest} = useApi(
 		{
-			description: 'Demo Key 2',
-			apiKey: 'txoediu8lvq01vzdpz38hldc',
-			createDate: 'Sep 07, 2023',
+			url: 'api/user/generate',
+			method: 'post',
+			data: {
+				keyDescription: inputValue,
+				apiKey:
+					Math.random()
+						.toString(RANDOM_STRING_LENGTH)
+						.substring(2, RANDOM_STRING_LENGTH) +
+					Math.random()
+						.toString(RANDOM_STRING_LENGTH)
+						.substring(2, RANDOM_STRING_LENGTH),
```

Comment:
We don't `apiKey` and `createDate` here, only description is enough, the frontend code was written to mock the behavior and there is no need to include them here